NeoMathEngine/src/CPU/CpuMathEngineDnn3dConv.cpp

```diff
@@ -440,12 +437,12 @@ void CCpuMathEngine::blob3dConvolutionBackward( const CCommon3dConvolutionDesc&
 
 			int outputLineEnd = outputLineStart + outputLineCount;
 			for( int step = outputLineStart; step < outputLineEnd; ++step ) {
-				CFloatHandle outputData = resultData + step * outputRowSize;
+				float* outputData = resultData + step * outputRowSize;
 
 				// Set the free term
 				if( freeTermData != 0 ) {
 					setVectorToMatrixRows( outputData, result.Width() * result.Depth(),
-						result.Channels(), *freeTermData );
+						result.Channels(), GetRaw( *freeTermData ) );
```

Comment:
`GetRaw` can be removed by using `const float* freeTermDataPtr = freeTermData == nullptr ? nullptr : GetRaw( *freeTermData )` initialized outside of `for( int b = batchStart ...`